R/pairwise-comparisons.R

```diff
@@ -1,24 +1,53 @@
-#' @title Do Pairwise Comparisons of Scores
+#' @title Obtain Pairwise Comparisons Between Models
 #'
 #' @description
 #'
-#' Compute relative scores between different models making pairwise
-#' comparisons. Pairwise comparisons are a sort of pairwise tournament where all
+#' Compare scores obtained by different models in a pairwise tournament. All
 #' combinations of two models are compared against each other based on the
 #' overlapping set of available forecasts common to both models.
-#' Internally, a ratio of the mean scores of both models is computed.
-#' The relative score of a model is then the geometric mean of all mean score
-#' ratios which involve that model. When a baseline is provided, then that
-#' baseline is excluded from the relative scores for individual models
-#' (which therefore differ slightly from relative scores without a baseline)
-#' and all relative scores are scaled by (i.e. divided by) the relative score of
-#' the baseline model.
-#' Usually, the function input should be unsummarised scores as
-#' produced by [score()].
-#' Note that the function internally infers the *unit of a single forecast* by
-#' determining all columns in the input that do not correspond to metrics
-#' computed by [score()]. Adding unrelated columns will change results in an
-#' unpredictable way.
+#'
+#' The input should be a `scores` object as produced by [score()]. Note that
+#' adding additional unrelated columns can unpredictably change results.
```

Comment:
Is there any particular reason we're worried about that here and/or is there a way that we can check it ourselves rather than telling the user?

Reply:
I think we can't check that - this would require storing the forecast unit at the time it was computed and comparing it with the forecast unit that we see now. 
There is nothing in the suggested workflows that would lead to this situation. But I can imagine users doing some computations on their scores and doing pairwise comparisons afterwards - that's why I thought the note would be helpful.

Reply:
In that case, can we just add why it can unpredictably change results?

Reply:
Updated to the following: 

"Note that adding additional unrelated columns can unpredictably change results, as all present columns are taken into account when determining the set of overlapping forecasts between two models"